bitnami/elasticsearch/README.md

```diff
@@ -70,6 +70,11 @@ The following table lists the configurable parameters of the Elasticsearch chart
 | `config`                                          | Elasticsearch node custom configuration                                                                                                                   | ``                                                           |
 | `extraVolumes`                                    | Extra volumes                                                                                                                                             |                                                              |
 | `extraVolumeMounts`                               | Mount extra volume(s),                                                                                                                                    |                                                              |
+| `initdbScripts`                                   | Dictionary of initdb scripts                                                                                                                              | `nil`                                                        |
+| `initdbScriptsConfigMap`                          | ConfigMap with the initdb scripts (Note: Overrides `initdbScripts`)                                                                                       | `nil`                                                        |
+| `extraEnvVars`                                    | Array containing extra env vars to be added to all pods (evaluated as a template)                                                                         | `[]`                                                         |
+| `extraEnvVarsConfigMap`                           | ConfigMap containing extra env vars to be added to all pods (evaluated as a template)                                                                     | `nil`                                                        |
```

Comment:
We need to come with a naming standard. In other chart we have CM instead of ConfigMap (probably because it is too long 😅 )

Reply:
OK, changed, although I still am more a fan of initdbScriptsConfigMap.